src/nft/components/collection/PriceRange.tsx

```diff
@@ -186,10 +187,11 @@ export const PriceRange = () => {
             top: '3px',
             width: '12px',
             height: '20px',
-            backgroundColor: `${theme.textPrimary}`,
+            opacity: '1',
+            backgroundColor: `white`,
             borderRadius: '4px',
             border: 'none',
-            boxShadow: `${theme.shallowShadow.slice(0, -1)}`,
+            boxShadow: colorsDark.shallowShadow.slice(0, -1),
```

Comment:
Why aren't we pulling this from theme anymore?

Reply:
theme has a different shadow color.  I couldn't find this exact shadow defined anywhere but in darktheme.